website/docs/language/import/index.mdx

```diff
@@ -0,0 +1,188 @@
+---
+page_title: Import - Configuration Language
+description: >-
+  Import resources into Terraform to bring them under Terraform's management by writing `import` blocks in code.
+---
+
+# Import
+
+-> **Note:** Import blocks are only available in Terraform v1.5.0 and later.
+
+~> **Experimental:** Import blocks are an experimental Terraform feature. While we do not expect to make backwards-incompatible changes to their syntax, processing of imports during the plan, generating configuration, and the `-generate-config-out` flag may change in future releases.
+
+Terraform can import existing infrastructure resources, which brings them under Terraform's management. Once you have imported a resource, it will be present in the Terraform configuration and state, just like a resource you had created via Terraform. You can then manage it like any other resource, updating its attributes and destroying it as part of a normal resource lifecycle.
```

Comment:
Thank you, this is much clearer. I'd like to avoid suggesting that the `import` block is doing the "importing" itself. Terraform plans and applies actions such as import, whereas code blocks like `import` only _declare_ what the user expects to have been done.

A practitioner, on the other hand, can be said to import resources _using_ Terraform. Perhaps:
```suggestion
Terraform can be used to import existing infrastructure resources, which brings them under Terraform's management. Once you have imported a resource, it will be present in the Terraform configuration and state, just like a resource you had created via Terraform. You can then manage it like any other resource, updating its attributes and destroying it as part of a normal resource lifecycle.
```